neo/Consensus/ConsensusService.cs

```diff
@@ -387,6 +369,28 @@ private void OnRecoveryMessageReceived(ConsensusPayload payload, RecoveryMessage
             }
         }
 
+        private void OnRecoveryRequestReceived(ConsensusPayload payload)
+        {
+            if (context.WatchOnly()) return;
+            if (!context.CommitSent())
+            {
+                bool shouldSendRecovery = false;
+                // Limit recovery to be sent from, at least, `f` nodes when the request is from a lower view number.
```

Comment:
I know that this code is moved, but the comment is not related to the code, when is checked the view number? Someone could explain me, what we try here?

Reply:
@shargon, I am going to clean this as done in PR707. Take a look.
In fact, the parameter of some function calls are not anymore being used.

Reply:
My friend, I updated the comment and removed the variable (directly using the `context.F()`).
This is an upper limit of those responding to a recovery request.

If someone is asking for recovery we consider it as faulty, thus, we remain with a max. number of `F-1` faulties. In this sense, we ask for `F`.

For an extra safety, we already suggested this limit to be increased to `F+1`.  However, we still did not found the case in which it will be really needed.
Thus, we are still keeping it to `F`.